components/infobox/extensions/commons/infobox_placement_stats.lua

```diff
@@ -0,0 +1,206 @@
+		local opponentConditions = Array.map(teamTemplates, function(teamTemplate)
+			return '[[opponenttemplate::' .. teamTemplate .. ']]'
+		end)
+
+		table.insert(conditions, '(' .. table.concat(opponentConditions, ' OR ') .. ')')
+	end
+
+	return table.concat(conditions, ' AND ')
+end
+
+---Fetches the placement count values for a given tier
+---@param tier string
+---@param baseConditions string
+---@param placementData InfoboxPlacementStatsData
+function PlacementStats._fetchForTier(tier, baseConditions, placementData)
+	placementData.tiers[tier] =  {top3 = 0, all = 0, placement = {}}
+
+	local queryData = mw.ext.LiquipediaDB.lpdb('placement', {
+		limit = 5000,
+		conditions = baseConditions .. ' AND [[liquipediatier::' .. tier .. ']]',
+		query = 'placement, count::placement',
+		groupby = 'placement asc'
+	})
+
+	for _, placement in pairs(queryData) do
+		local count = tonumber(placement.count_placement)
+		local place = tonumber(mw.text.split(placement.placement or '', '-', true)[1])
+
+		placementData.tiers[tier].all = placementData.tiers[tier].all + count
+		placementData.totals.all = placementData.totals.all + count
+		if place and place <= 3 then
+			placementData.tiers[tier].placement[place] = (placementData.tiers[tier].placement[place] or 0) + count
+			placementData.tiers[tier].top3 = placementData.tiers[tier].top3 + count
+			placementData.totals.placement[place] = (placementData.totals.placement[place] or 0) + count
+			placementData.totals.top3 = placementData.totals.top3 + count
+		end
+	end
+end
+
+---Builds the display
+---@param placementData InfoboxPlacementStatsData
+---@return Html
+function PlacementStats._buildTable(placementData, tiers)
+	local display = mw.html.create('table')
+		:addClass('wikitable sortable wikitable-striped wikitable-bordered')
+		:css('text-align', 'center')
+		:node(PlacementStats._header())
+
+	for _, tier in ipairs(tiers) do
+		display:node(PlacementStats._buildRow(placementData.tiers[tier], tier))
+	end
+
+	display:node(PlacementStats._buildBottom(placementData))
+
+	local infoboxHeader = mw.html.create('div')
+		:tag('div')
+			:addClass('infobox-header wiki-backgroundcolor-light')
+			:wikitext('Placement Summary')
+			:done()
+
+	return mw.html.create('div')
+		:addClass('fo-nttax-infobox-wrapper')
+		:tag('div')
+			:addClass('fo-nttax-infobox wiki-bordercolor-light')
+			:node(infoboxHeader)
+			:node(display)
+			:done()
+end
+
+---Builds the header
+---@return Html
+function PlacementStats._header()
+	return mw.html.create('tr')
+		:tag('th'):wikitext('Tier'):css('text-align', 'left'):css('width', '100%'):done()
+		:tag('th'):wikitext(Medal['1']):done()
+		:tag('th'):wikitext(Medal['2']):done()
+		:tag('th'):wikitext(Medal['3']):done()
+		:tag('th'):wikitext(Abbreviation.make('Top3', 'Total of top 3')):done()
+		:tag('th'):wikitext(Abbreviation.make('All', 'Only count those appear in tournaments prizepool sections.')):done()
```

Comment:
What is the intention of this abbreviation, is it to hint that there might be tournaments where the opponent played but isn't listed in the prizepool? I don't really think it is needed.